### PR TITLE
gen-block script update to account for new MD format

### PR DESCRIPTION
**Pull request by zarbod**
Changed the gen-block script to the new format.

### CODE REVIEW

**Comment by zarbod** [comment]
I had to do a bit of refactoring. I had no idea we moved the code to a new directory. Is there a reason for this?

**Comment by sanketgarade** [comment]
> I had to do a bit of refactoring. I had no idea we moved the code to a new directory. Is there a reason for this? 


Yes I rearranged the folders to keep all the **browse site** related **code** and the **markdown files** in a single folder. 

Similarly the **search site** front end will be kept in a separate folder in the root path. 

Sorry if I didn't update you on this, but as a general rule make sure you always pull the upstream code before starting to do any changes at your end. Cause I may do some minor changes every now and then on the main repo. 

And thanks for this PR. I'll check it in detail later today.

**Comment by sanketgarade** [comment]
@zarbod your PR contains the `src` folder in the root path. But all scripts have been moved to `/browse/scripts`. Can you also pls update this PR with the same and test it? Once that is ok, Ill merge it. Make sure that the `src` folder is removed from the root path. (refer the current HEAD on `develop` branch)

File: src/filterdb.py [hunk 1]
```diff
@@ -0,0 +1,82 @@
+import csv
```
Comment (sanketgarade) [inline src/filterdb.py:1]: all files in the src folder to be moved to the browse/scripts folder and the src folder to be removed.

**Review by sanketgarade — CHANGES_REQUESTED** [review]

File: src/filterdb.py (hunk 1 above)
Comment (sanketgarade) [reply src/filterdb.py:1]: fixed

**Comment by sanketgarade** [comment]
@zarbod hey I fixed this at my end and tested it. Merged into develop. Thanks a lot!!
I did this on priority since I want to create the markdown files for the browse site.

**Pull request merged**